src/graphql/mutations/CreateCategory.js

```diff
@@ -0,0 +1,40 @@
+import { GraphQLString, GraphQLNonNull } from 'graphql';
+
+import { assertUser } from 'graphql/util';
+
+import client from 'util/client';
+
+import MutationResult from 'graphql/models/MutationResult';
+
+export default {
+  type: MutationResult,
+  description: 'Create a category',
```

Comment:
Do we need a category creation API? I thought we only allow users to add existing categories to articles for now.

Reply:
Since we got some feedbacks from Flow, and we already added the ~4 additional categories. So I think we can have that API, and only use by team or AI model ?

create the issue for the discussion https://github.com/cofacts/rumors-api/issues/158

Reply:
Thanks for explanation. Let's resolve this conversation for now and keep discussion in the ticket.